charts/pega/charts/installer/templates/_pega-installer-job.tpl

```diff
@@ -34,7 +41,11 @@ spec:
       - name: {{ template "pegaVolumeInstall" }}
         configMap:
           # This name will be referred in the volume mounts kind.
+     {{- if (eq .root.Values.global.actions.execute "install") }}
           name: {{ template "pegaInstallConfig"}}
+     {{- else }}
+          name: {{ template "pegaUpgradeConfig"}}
+     {{- end }}
```

Comment:
This could break the use case where `actions.execute` is `install-deploy` and we end up having `pega-upgrade-config` as configMap. Though it is just a naming, but it adds confusion to the user.